conf/nomad/trino.hcl

```
@@ -218,8 +218,17 @@ ${hive_config_properties}
 EOH
         destination = "secrets/trino/hive.properties"
       }
+%{ if use_memory_connector }
       template {
-        # TODO: Create issue with hashicorp. Is there a way to mount directly to /etc/hosts ( for continual updates )
+        destination = "local/trino/catalog/memory.properties"
+        data = <<EOH
+connector.name=memory
+memory.max-data-per-node=${max_data_per_node}
```

This variable must be named to fit as variables are global.
```suggestion
memory.max-data-per-node=${connector_memory_max_data_per_node}
```

---

I'm totally down to have a better suited name, but what exactly do you mean by "variables are global", and how does that necessitate the change?

---

`max_data_per_node` _sounds_ like a Trino core variable. However it is a memory connector specific variable.

---

Ahh, that is a very good point. Thanks